ckanext/validation/tests/test_form.py

```diff
@@ -405,8 +405,8 @@ def test_resource_form_create_invalid(self, app):
         )
 
         assert "validation" in response.body
-        assert "missing-value" in response.body
-        assert "Row 2 has a missing value in column 4" in response.body
+        assert "missing-cell" in response.body
+        assert "This row has less values compared to the header row" in response.body
```

Comment:
Is there somewhere in the new error messages that says what row and column are the failing ones (ie 2 and 4)?

Reply:
I changed the tested with the row and field information. The format is different from the goodtables and it has the character '"' escaped inside the html.